Source/JavaScriptCore/runtime/JSCBytecodeCacheVersion.cpp

```diff
@@ -45,7 +47,7 @@ static constexpr bool verbose = false;
 
 uint32_t computeJSCBytecodeCacheVersion()
 {
-#if OS(DARWIN)
+#if OS(DARWIN) && USE(APPLE_INTERNAL_SDK)
```

Comment:
These changes are not necessary after https://github.com/WebKit/WebKit/commit/815968c690eb9551f3378bbdd996c624ea1289ac